src/components/misc/ProgressBar.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class ProgressBar extends React.Component {
+    static propTypes = {
+        progressSum: React.PropTypes.number.isRequired,
```

Comment:
Why "`progressSum`" and not just `progress`? This isn't really a sum (it's a fraction or a product), and it seems unnecessary to make the property name longer than necessary.

Actually, I would probably make this a fraction between 0-1, and multiply by 100 inside the component, instead of, like now, multiplying by 100 and passing the multiplied value as a property.